api/HTMLInputElement.json

```diff
@@ -1186,6 +1186,7 @@
       },
       "max": {
         "__compat": {
+          "spec_url": "https://html.spec.whatwg.org/multipage/input.html#the-min-and-max-attributes",
```

Comment:
Given this is about the IDL properties of the `HTMLInputElement` class and not about the attributes in the HTML `<input>` tag, I wonder if https://html.spec.whatwg.org/multipage/input.html#dom-input-max would be the correct specification pointer here. CC'ing @sideshowbarker for ... _input_ :)

Reply:
Thanks for your "input"! Moved all links to the correct place.